src/fondant/component_spec.py

```diff
@@ -191,14 +192,22 @@ def outputs_additional_subsets(self) -> bool:
 
     @property
     def args(self) -> t.Mapping[str, Argument]:
+        def _get_default(argument_info):
+            if "default" in argument_info:
+                if argument_info["default"] == "None":
```

Comment:
This is needed for arguments that are taken from the componentSpec which always have a string type